nerdlets/e2m-gui-nerdlet/form-components/account-selector.js

```diff
@@ -1,45 +1,48 @@
-import React from "react";
-import PropTypes from "prop-types";
-
-import SelectSearch from "react-select-search";
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Spinner } from 'nr1';
```

Comment:
Previous Spinner?

Reply:
The code may not have loaded b/c it was referenced in the code. If it's never executed, there's a chance that the inevitable error never occurred.